### PR TITLE
VIAMW-198 Collection Page

### PR DESCRIPTION
**Pull request by Erikbohlmark**

### CODE REVIEW

File: src/main/java/tv/accedo/via/middleware/appgrid/tools/changeset/update1_3/CollectionPage.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    private List<String> getAllPlatformIds(String applicationGroupId){
+
+        // Get all platformIds from appgrid for the application group.
+        return getPlatformsByApplicationGroup(applicationGroupId).stream()
+                                                                 .map(Platform::getId)
+                                                                 .collect(Collectors.toList());
+    }
+
+    private EntryType getCollectionPageStruction(String accountId, String applicationGroupId, List<String> platformIds) throws IOException {
```
Comment (AlinStoianAccedo) [inline src/main/java/tv/accedo/via/middleware/appgrid/tools/changeset/update1_3/CollectionPage.java]: What's a "struction" :) ?

File: src/main/java/tv/accedo/via/middleware/appgrid/tools/changeset/update1_3/SortByAndOrder.java [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
 import java.util.stream.Collectors;
 
 @Slf4j
-public class ChangeSet_1_3 extends AbstractChangeSet {
+public class SortByAndOrder extends AbstractChangeSet {
 
     @Override
     public void run(){
-        String applicationGroupId = ApplicationGroupType.VIA_GO_STAGING.getApplicationGroupId();
+        String applicationGroupId = System.getProperty(ChangeSetOptions.APPLICATION_GRUOP_ID);
```
Comment (AlinStoianAccedo) [inline src/main/java/tv/accedo/via/middleware/appgrid/tools/changeset/update1_3/SortByAndOrder.java]: Maybe create a getProperty method in AbstractChangeSet that also prints a message if the param is missing:
```
       if(appGroupId == null){
            throw new IllegalArgumentException("Missing argument: -Dappgroupid");
        }
```

Comment (Erikbohlmark) [inline src/main/java/tv/accedo/via/middleware/appgrid/tools/changeset/update1_3/SortByAndOrder.java]: true

**Comment by Erikbohlmark** [comment]
I have now updated with a validation check on application group.

**Pull request merged**